drivers/volume/portworx/portworx.go

```diff
@@ -1050,6 +1057,30 @@ func (d *portworx) UpgradeDriver(version string) error {
 	return nil
 }
 
+// GetClusterPairingInfo return underlying storage details
+func (d *portworx) GetClusterPairingInfo() (map[string]string, error) {
+	pairInfo := make(map[string]string)
+	pxNodes, err := d.schedOps.GetRemotePXNodes(remoteKubeConfigPath)
+	if err != nil {
+		logrus.Errorf("err retriving remote px nodes: %v", err)
+		return pairInfo, err
+	}
+
+	clusterMgr, err := d.getClusterManagerByAddress(pxNodes[0].Addresses[0])
+	resp, err := clusterMgr.GetPairToken(true)
```

Comment:
true will reset the token every time. You shouldn't do that.

Reply:
default value should be false then?

Reply:
Yes